bigquery/bqml/requirements.txt

```diff
@@ -3,6 +3,6 @@ google-cloud-bigquery-storage==2.14.0
 pandas==1.1.5; python_version < '3.7'
 pandas==1.3.5; python_version > '3.6'
 # pyarrow must stay under 7
```

Comment:
Is this still relevant? If so, that might explain the test failures.

Reply:
Good catch!